### DISCUSSION TITLE
Ubuntu: How to launch a mame version you compiled yourself through the terminal?

### DISCUSSION THREAD

**Discussion by HazukiXX** [mamedev/discussions#144] [Q&A] [answered]
I compiled a nightly version of mame but when I open a terminal in the folder where the executable is and run "mame", I get this error: 
![Untitled](https://github.com/user-attachments/assets/1fd164fe-2b9e-4b19-b677-83bad3b0a247)

Doing "sudo apt install mame" will give you some outdated version of the emulator that's in the repo of your respective distro (example: 0.242 for Ubuntu Jammy), and running "mame" will run that version instead of the version you compiled, even if run it inside your compiled version's folder.

So how can I launch a self-compiled mame through the terminal?

**Comment by einstein95** [comment, marked as answer]
Either move the executable to a place in `$PATH` or run it like `./mame`

**Reply by HazukiXX** [reply]
Thank you, that worked.
